projects/ccd-case-ui-toolkit/src/lib/shared/services/fields/fields.purger.ts

```diff
@@ -309,10 +309,16 @@ export class FieldsPurger {
    * @returns A new array with the mapped values
    */
   public mapArrayValuesToNull(array: any[]): any[] {
-    return array.map(element => {
-      return typeof element === 'object'
-        ? Object.assign({}, ...Object.keys(element).map(k => ({ [k]: null })))
-        : null;
-    });
+    if (array && array.length > 0) { FieldsUtils.isArray([])
```

Comment:
@OgunyemiO why you have added FieldsUtils.isArray([]?

Reply:
@RiteshHMCTS Oops! I was trying something out and mistakenly left it. :) I will remove now.